### ISSUE TITLE
What is version stable? i have upgrade/install Teampass 3.1.1 and contain erros to fix

### ISSUE THREAD

**Issue by sysadminketra** [nilsteampassnet/TeamPass#4441]
## i have install/upgrade Teampass 3.1.1 and contain erros to fix
I recently installed TeamPass version 3.1.1, expecting it to be stable, but I encountered several issues affecting functionality. Could you clarify which version is currently recommended as stable for production use? 
Admin User Permissions: The admin user cannot change the status of other users through the admin panel, which restricts user management tasks.
Database Schema Issues: There are several columns in the database tables that appear to require adjustments or fixes to function properly.

### Server configuration
**Operating system**:

| Sistema | from: (versão) | to: (versão) |
|--|--|--|
| Sistema Operativo |  Debian V9 | Debian V10 - V11  |
| PHP | V7.4 | V8.1 |
| MariaDB | V10.1 | V 10.7 |
| Teampass | 2.1.27.23 | 3.1.1 latest |

Thank you for your guidance and support in identifying a stable release.

**Comment by DobroFenix** [comment]
There is no stable version yet.
You can run it from this repository, but no one guarantees its stability.
I myself am waiting for the release of a stable assembly.

**Comment by DaveWebb2** [comment]
Agreed, I've been trying to upgrade from version 2.1.27.36 to version 3 for over 6 months and haven't been able to get a version that works.

**Comment by DobroFenix** [comment]
Making a clean installation of different versions from different 3.0 branches, I also could not fully make the system work.
Problems begin immediately with the installation stage, which the installer has to be modified to complete the installation of all stages.
Most of the functionality does not work for the installation, personal folders and cron do not help.
I am still waiting for the release of the new version, which I hope will be working.

**Comment by DaveWebb2** [comment]
I have been trying to upgrade from version 2 for about a year now, and have not been able to get version 3 working to a point that we can actually use it.  I've tried upgrades and fresh installs, there are always problems, it never 'just works'.
